test/assets/helm-charts/keptn/values-int-tests-dev.yaml

```diff
@@ -5,3 +5,7 @@ global:
 shipyardController:
   gracePeriod: 10
   preStopHookTime: 5
+
+nats:
+  podAnnotations:
+    cluster-autoscaler.kubernetes.io/safe-to-evict: "true"
```

Comment:
why this is necessary now?

Reply:
this just enables the cluster autoscaler to move nats pods around during node down-scaling. So it just prevent some auto scaling issues that i found